From 2261d0e4ee6a7f639ea2c0bae2496e4e7c773824 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Felix=20Paul=20K=C3=BChne?= <fkuehne@videolan.org>
Date: Thu, 14 Jan 2016 13:10:15 +0100
Subject: [PATCH 20/20] contrib/live555: update version and fetch from v.o
 instead

Upstream removes the current tarball on every update and therefore breaks VLC's compilation, which is extremely annoying

(cherry picked from commit 2929dada4d63cc865968c2f0d2ab801ee2e24c8a)
Signed-off-by: Jean-Baptiste Kempf <jb@videolan.org>
---
 contrib/src/live555/SHA512SUMS | 2 +-
 contrib/src/live555/rules.mak  | 6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/contrib/src/live555/SHA512SUMS b/contrib/src/live555/SHA512SUMS
index 5fd9575..5410ef7 100644
--- a/contrib/src/live555/SHA512SUMS
+++ b/contrib/src/live555/SHA512SUMS
@@ -1 +1 @@
-e5cec29b769e8c10dcc932c1ed96f88d216a3643b2301d36d8df58b8db9bf823247876f2b4cbccac0fcb61ae098eebff3df3b767b1710e5b8b279ffc3bec4eb1  live.2015.12.22.tar.gz
+0183f71638919eb092a5ddf7ccff59181b7bcd6fdd5ce18509f357a3a830e6f98435434bf3d9cc66c951cc62c4e817ec1775200933e718d46ef463b9ad1f8b24  live.2016.01.12.tar.gz
diff --git a/contrib/src/live555/rules.mak b/contrib/src/live555/rules.mak
index 87f0847..4babae5 100644
--- a/contrib/src/live555/rules.mak
+++ b/contrib/src/live555/rules.mak
@@ -1,8 +1,8 @@
 # live555
 
-LIVE555_FILE := live.2015.12.22.tar.gz
-LIVEDOTCOM_URL := http://live555.com/liveMedia/public/$(LIVE555_FILE)
-#LIVEDOTCOM_URL := $(CONTRIB_VIDEOLAN)/live555/$(LIVE555_FILE)
+LIVE555_FILE := live.2016.01.12.tar.gz
+#LIVEDOTCOM_URL := http://live555.com/liveMedia/public/$(LIVE555_FILE)
+LIVEDOTCOM_URL := $(CONTRIB_VIDEOLAN)/live555/$(LIVE555_FILE)
 
 ifdef BUILD_NETWORK
 PKGS += live555
-- 
2.7.0

